src/views/DandisetLandingView/DandisetOwners.vue

```diff
@@ -0,0 +1,172 @@
+      </v-card-subtitle>
+      <template v-if="!owners || !owners.length">
+        <v-row
+          align="center"
+          justify="center"
+          class="mx-1"
+        >
+          No owners
+        </v-row>
+      </template>
+      <template v-else>
+        <v-tooltip
+          top
+          :disabled="userCanModifyDandiset"
+        >
+          <template #activator="{ on }">
+            <div v-on="on">
+              <v-row class="mx-1 px-1">
+                <v-autocomplete
+                  v-model="selection"
+                  :items="items"
+                  :disabled="!userCanModifyDandiset"
+                  :loading="loadingUsers"
+                  :search-input.sync="search"
+                  hide-no-data
+                  clearable
+                  auto-select-first
+                  item-text="result"
+                  placeholder="enter email address"
+                  outlined
+                  flat
+                  return-object
+                  @update:search-input="throttledUpdate"
+                />
+              </v-row>
+            </div>
+          </template>
+          <template v-if="loggedIn">
+            You must be an owner to manage ownership.
+          </template>
+          <template v-else>
+            You must be logged in to manage ownership.
+          </template>
+        </v-tooltip>
+        <v-row class="mx-1">
+          <v-chip
+            v-for="(owner, i) in newOwners"
+            :key="i"
+            color="light-blue lighten-4"
+            text-color="light-blue darken-3"
+            class="font-weight-medium ma-1"
+            :close="userCanModifyDandiset"
+            @click:close="removeOwner(i)"
+          >
+            {{ owner.name || owner.username }}
+          </v-chip>
+        </v-row>
+      </template>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import {
+  defineComponent, reactive, ref, computed, Ref, ComputedRef, watch,
+} from '@vue/composition-api';
+
+import { debounce } from 'lodash';
+
+import { publishRest, loggedIn as loggedInFunc } from '@/rest';
+import { User, Version } from '@/types';
+
+interface UserResult extends User {
+  result: string
+}
+
+// Includes a field `result` on each user which is the value displayed in the UI
+const appendResult = (users: User[]): UserResult[] => users.map((u: User) => ({ ...u, result: (u.name) ? `${u.name} (${u.username})` : u.username }));
+
+export default defineComponent({
+  name: 'DandisetOwners',
+  props: {
+    userCanModifyDandiset: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  setup(props, ctx) {
+    const search: Ref<string> = ref('');
+    const loadingUsers: Ref<boolean> = ref(false);
+    const selection: Ref<UserResult|null> = ref(null);
+    const items: Ref<UserResult[]> = ref([]);
+
+    const store = ctx.root.$store;
+
+    const currentDandiset: ComputedRef<Version> = computed(
+      () => store.state.dandiset.publishDandiset,
+    );
```

Comment:
```suggestion
    const currentDandiset = store.state.dandiset.publishDandiset;
```
I'm pretty sure direct-vuex uses all the reactive magic, so you can just grab things out of it directly and it will infer the type automatically.